tests/integration/nm/ovs_test.py

```diff
@@ -132,6 +131,7 @@ def _get_bridge_current_state():
             for dev in nm.device.list_devices()
         ]
         state = nm.ovs.get_bridge_info(nmdev, devices_info)
+    nm.nmclient.delete_client()
```

Comment:
Why is the deletion moved to the end? Why can it not replace the refresh workaround?

Reply:
Because if running at the middle of the process, other function might still have reference to the NM.Client.